pkg/README.md

```diff
@@ -0,0 +1,3 @@
+# Standalone packages
+
+The `pkg` directory (the current directory) contains a set of standalone packages that can be imported and used by any applications. The packages are specifically meant for the development of the Mainflux based back-end applications and implement common tasks needed by the programmatic operation of Mainflux platform.
```

Comment:
`by any application`

Reply:
Also, as a reference, maybe you can take a look here at the explanation of the role of this standard directory: https://github.com/golang-standards/project-layout/tree/master/pkg